lib/rust/flexer/src/automata/nfa.rs

```diff
@@ -286,10 +297,10 @@ pub mod tests {
                 State::from(vec![7,11]).named("group0_rule1"),
                 State::default(),
             ],
-            alphabet: Alphabet::from(vec![32,33,97,123]),
+            alphabet_segmentation: alphabet::Segmentation::from_divisions(vec![32, 33, 97, 123].as_slice()),
```

Comment:
2 things:

1. spaces after `,` 
2. instead of `vec![32, 33, 97, 123].as_slice()` just write `&[32,33,97,123]` :)